src/components/Kv/KvSettingsCard.vue

```diff
@@ -1,16 +1,18 @@
 <template>
-	<div class="settings-card" :class="{ obscure: disabled }">
-		<div class="settings-card__icon-wrapper">
-			<!-- @slot Icon of the settings card. Should not be empty. -->
-			<slot name="icon"></slot>
-		</div>
-		<div class="settings-card__content-wrapper">
-			<div class="settings-card__title">
-				<h3>{{ title }}</h3>
+	<div class="row">
```

Comment:
This is a more broad point, but in general I like to keep classes that set the width and margin of the whole component outside of re-usable components, especially Kv* components.

We don't always know how a Kv*Component will be re-used, so letting the parent component handle margins and widths gives more flexibility. In this example, we may want a KvSettingsCard to span 12 columns in some other layout.

Reply:
I could go either way on this, but you've convinced me, I'll change it back.